### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by DerGoogler**
Using new GitHub Issue forms. 

(@Atrate)

### CODE REVIEW

**Comment by Atrate** [comment]
Could you enable issues in your repo so I can test them out?

**Comment by DerGoogler** [comment]
@Atrate,

I have enabled issues, you can try it now. 

> Sorry for the late reply

File: .github/ISSUE_TEMPLATE/module-submission.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    id: module-description
+    attributes:
+      label: Description
+      placeholder: A simple module that does this, that, and the other thing.
+    validations:
+      required: true
+  - type: input
+    id: module-link
+    attributes:
+      label: Link
```
Comment (Atrate) [inline .github/ISSUE_TEMPLATE/module-submission.yml:40]: `label: Link` -> `label: Module repository link`

File: .github/ISSUE_TEMPLATE/module-submission.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      placeholder: https://www.github.com/username/repo
+    validations:
+      required: true
+  - type: input
+    id: module-source-code
+    attributes:
+      label: Link
```
Comment (Atrate) [inline .github/ISSUE_TEMPLATE/module-submission.yml:47]: `label: Link` -> `label: Source code link (for compiled binaries/APKs)`

File: .github/ISSUE_TEMPLATE/module-submission.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      placeholder: None
```
Comment (Atrate) [inline .github/ISSUE_TEMPLATE/module-submission.yml:48]: `placeholder: None` -> `placeholder: https://www.github.com/username/repo`

**Review by Atrate — CHANGES_REQUESTED** [review]

**Comment by DerGoogler** [comment]
@Atrate , fixed

**Review by Atrate — APPROVED** [review]

**Comment by Atrate** [comment]
@DerGoogler Thanks!

**Pull request merged**